src/19-ipfs-0-4-3-released/index.md

```diff
@@ -0,0 +1,48 @@
+---
+date: 2016-09-20
+id: 19-ipfs-0-4-3-released
+template: tmpl/layouts/post.html
+baseurl: ..
+breadcrumbs:
+- {name: "19-ipfs-0-4-3-released", link: "./" }
+tags: modules
+title: go-ipfs 0.4.3 has been released
+author: Lars Gierth
+collection: posts
+---
+
+[go-ipfs 0.4.3](https://dist.ipfs.io/#go-ipfs) has been released today,
+and we're incredibly proud as it's the fastest and most stable IPFS ever.
+Give it a try as soon as you can, we're sure you'll like it as much as we do.
+
+## What's changed?
+
+- Runtime performance has improved all over the place!
+  Bitswap, Content Routing and Peer Routing, Datastore, Blockstore, you name it.
+- Memory usage has improved as Provider records are now stored on disk.
+- We now use Golang 1.7, which produces smaller and faster binaries.
+- Connectivity between nodes is stabler than ever,
+  as we fixed transport issues that were previously thought to be NAT-related.
+- The go-ipfs daemon now automatically raises the limit of file descriptors if needed.
+- The daemon now automatically runs fs-repo migrations.
+- The daemon got new flags:
+  - `ipfs daemon --offline` can be used to disable all swarm networking.
+  - `ipfs daemon --migrate` automatically runs any pending fs-repo migrations.
+- The Datastore got a few more config options:
+  - `Datastore.HashOnRead` allows you to verify data on read access.
+  - `Datastore.BloomFilterSize` tunes the new lookup cache.
+
+And plenty of bug fixes as well as improvements to documentation and test coverage.
+[Take a look at the Changelog](https://github.com/ipfs/go-ipfs/blob/master/CHANGELOG.md).
```

Comment:
maybe we should include the changelog here? that way people get it on their email and they can read it right there